src/react-native/13-making-http-requests/making-http-requests.md

```diff
@@ -244,8 +238,16 @@ It will be helpful to have a third terminal tab for the `npm run api` command.
 - Update the `useState` in `hooks.ts` to call `useEffect()` and `fetch` data from `${process.env.PMO_API}/states`.
 - Update `StateList.tsx` to call `useState()` and use the `StateResponse` interface.
 
+✏️ Duplicate **.env.example** to **.env** in your project.
```

Comment:
I think the idea of moving this down was to tell them to do it in the list above, not have it be a “step” (pencil icon).